src/components/layout.tsx

```diff
@@ -45,6 +45,13 @@ const Layout = ({ children }: LayoutProps) => {
           address {
             text
           }
+          sheltertech_logo_link {
```

Comment:
It isn't at all a priority, but I do agree it would be nice if we could encapsulate this query somehow. Maybe we can throw a ticket in the backlog for investigating that?

Reply:
Yep, good call. Created a ticket to capture that: https://github.com/ShelterTechSF/sheltertech.org/issues/275